engines/pyfunc-ensembler-job/environment.yaml

```diff
@@ -1,6 +1,6 @@
 name: pyfunc-ensembler-job
 dependencies:
-  - python=3.8
+  - python=3.7
```

Comment:
I think we need to update these for the engines too: https://github.com/gojek/turing/blob/main/engines/pyfunc-ensembler-job/setup.py#L25